client/my-sites/email/titan-setup-thank-you/theme.ts

```diff
@@ -0,0 +1,37 @@
+export type ThankYouThemeProps = { theme?: ThankYouThemeType };
+export type ThankYouThemeType = {
+	colors: { textColorLight: string; borderColorLight: string; background: string };
+	breakpoints: {
+		desktopUp?: string;
+		tabletUp?: string;
+		bigPhoneUp?: string;
+		smallPhoneUp?: string;
+		tabletDown: string;
+	};
+	weights: {
+		bold: string;
+		normal: string;
+	};
+	fontSize: {
+		small: string;
+	};
+};
```

Comment:
Do we really need to introduce those types which duplicate [those ones](https://github.com/Automattic/wp-calypso/blob/df370f92cad143abc77ef605c1c7e7fac2b55334/client/components/thank-you/types.ts#L36-L54)? It looks like the only difference is that the `studioGrey` property was renamed to `background`.

Reply:
I've reworked the theme object, it should look now clearer and better 🙂